modules/src/applications/transfer/msgs/transfer.rs

```diff
@@ -23,7 +24,7 @@ pub struct MsgTransfer {
     /// the channel by which the packet will be sent
     pub source_channel: ChannelId,
     /// the tokens to be transferred
-    pub token: Option<ibc_proto::cosmos::base::v1beta1::Coin>,
+    pub token: IbcCoin,
```

Comment:
I don't think denoms of the form `"ibc/{hash}"` are ever supposed to be sent over the wire, right? They're only an ibc-go internal detail IIUC.